Classes/Plugin/Search.php

```diff
@@ -551,6 +551,27 @@ public function makeFacetsMenuArray($content, $conf) {
         $selectQuery = $solr->service->createSelect($search['params']);
         $results = $solr->service->select($selectQuery);
         $facet = $results->getFacetSet();
+
+        $facetCollectionArray = array();
+
+        // replace everything expect numbers and comma
+        $facetCollections = preg_replace('/[^0-9|,]/', '', $this->conf['facetCollections']);
```

Comment:
This regular expression would allow `|` to be present in the string as well. Please remove the `|` to only allow numbers and comma.
```suggestion
        $facetCollections = preg_replace('/[^0-9,]/', '', $this->conf['facetCollections']);
```